js/localization.js

```diff
@@ -103,7 +103,11 @@ function tNodeList(nodes) {
 
 function tDocLoader() {
   t.DOMParser = new DOMParser();
-  t.cache = tryJSONparse(localStorage.L10N) || {};
+  try {
```

Comment:
In some js engines try-catch may still cause deopt of the entire function and its nested sub-functions. Use an IIFE: `(() => { ...... })();`